Example/ViewController.swift

```diff
@@ -18,6 +18,8 @@ class ViewController: UIViewController {
     
     var trackStyledFeature: StyledFeature!
     var rawTrackStyledFeature: StyledFeature!
+    var speedLimitView: SpeedLimitView!
+    var speedLimitViewPosition: MapOrnamentPosition = .topTrailing
```

Comment:
The position is hard-coded and seems to be constant, so we don’t need to support both leading and trailing positions in this example.

Reply:
Don't we need to support leading and trailing in order to support both LTR and RTL languages?

Reply:
A leading constraint will automatically flip in right-to-left languages.

Reply:
This variable is now unused, so we can remove it.